Makefile

```diff
@@ -80,6 +80,7 @@ fix-copies:
 	python utils/check_copies.py --fix_and_overwrite
 	python utils/check_table.py --fix_and_overwrite
 	python utils/check_dummies.py --fix_and_overwrite
+	python utils/check_task_guides.py --fix_and_overwrite
```

Comment:
The same should be added in `make repo-consistency`, without the `--fix_and_overwrite`.